projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+  "block.minecraft.banner.rats.rat_and_crossbones.pink": "粉红色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.purple": "紫色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.red": "红色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.white": "白色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_crossbones.yellow": "黄色老鼠与交叉骨",
+  "block.minecraft.banner.rats.rat_and_sickle.black": "黑色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.blue": "蓝色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.brown": "棕色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.cyan": "青色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.gray": "灰色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.green": "绿色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.light_blue": "淡蓝色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.light_gray": "淡灰色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.lime": "黄绿色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.magenta": "品红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.orange": "橙色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.pink": "粉红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.purple": "紫色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.red": "红色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.white": "白色老鼠和镰刀",
+  "block.minecraft.banner.rats.rat_and_sickle.yellow": "黄色老鼠和镰刀",
+  "block.rats.air_raid_siren": "防空警报",
+  "block.rats.auto_curdler": "凝乳台",
+  "block.rats.auto_curdler.desc0": "自动化奶酪生产",
+  "block.rats.auto_curdler.desc1": "§o“奶进，奶酪出”",
+  "block.rats.black_marbled_cheese": "黑色大理石纹奶酪",
+  "block.rats.block_of_blue_cheese": "蓝纹奶酪块",
+  "block.rats.block_of_cheese": "奶酪块",
+  "block.rats.block_of_nether_cheese": "熔岩奶酪块",
+  "block.rats.brain_block": "脑方块",
+  "block.rats.cauldron_blue_cheese": "蓝纹奶酪炼药锅",
+  "block.rats.cauldron_cheese": "奶酪炼药锅",
+  "block.rats.cauldron_milk": "牛奶炼药锅",
+  "block.rats.cauldron_nether_cheese": "熔岩奶酪炼药锅",
+  "block.rats.cheese_ore": "奶酪矿石",
+  "block.rats.chunky_cheese_token": "肥厚奶酪令牌",
+  "block.rats.chunky_cheese_token.desc0": "极·其·罕·见的老鼠掉落物",
+  "block.rats.chunky_cheese_token.desc1": "随处右击可以打开通往鼠西洲的传送门",
+  "block.rats.compressed_garbage": "压缩垃圾",
+  "block.rats.compressed_garbage.desc": "和普通垃圾堆相比会生成两倍数量的老鼠",
+  "block.rats.compressed_rat": "鼠套块",
+  "block.rats.compressed_rat.desc": "§o“多亏我们生而逢时，只因兴致如同美妙的果实，如若结出还需将渴求节制，腐烂吧！瓜熟蒂落的点子。——来自神秘视界生物融合看板”",
+  "block.rats.cursed_garbage": "有害垃圾",
+  "block.rats.cursed_garbage.desc": "有害垃圾上会生成疫鼠",
+  "block.rats.dutchrat_bell": "飞翔的荷兰鼠的钟",
+  "block.rats.dye_sponge": "染料海绵",
+  "block.rats.dye_sponge.desc": "将老鼠或羊身上的染料去除",
+  "block.rats.fish_barrel": "鱼桶",
+  "block.rats.garbage_pile": "垃圾堆",
+  "block.rats.garbage_pile.desc": "垃圾堆上会生成老鼠",
+  "block.rats.jack_o_ratern": "鼠瓜灯",
+  "block.rats.manhole": "井盖",
+  "block.rats.marbled_cheese": "大理石纹奶酪",
+  "block.rats.marbled_cheese_brick": "大理石纹奶酪砖块",
```

```suggestion
  "block.rats.marbled_cheese_brick": "大理石纹奶酪砖",
```
请参这篇提到“砖”与“砖块”区分方法的文章。此处应按原版“石砖”类，译作……砖
[翻译用语共识.md](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/e48f398f3b4181d39ea109693210740cb17528dd/CFPA-docs/%E7%BF%BB%E8%AF%91%E7%94%A8%E8%AF%AD%E5%85%B1%E8%AF%86.md)